addons/hr_expense/static/tests/mock_server/mock_models/hr_expense.js

```diff
@@ -0,0 +1,25 @@
+import { models } from "@web/../tests/web_test_helpers";
+
+export class HrExpense extends models.ServerModel {
+    _name = "hr.expense";
+
+    get_expense_dashboard() {
```

Comment:
Do we really want to hard code this specific stuff in the model itself?

Reply:
I don't think it's a big deal right now. it can be cleaned when it's actually useful to clean.